x-pack/legacy/plugins/maps/public/layers/styles/vector/components/field_select.js

```diff
@@ -7,18 +7,21 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import { EuiComboBox, EuiHighlight } from '@elastic/eui';
+import { EuiComboBox, EuiHighlight, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { FIELD_ORIGIN } from '../../../../../common/constants';
 import { i18n } from '@kbn/i18n';
 import { FieldIcon } from '../../../../../../../../../src/plugins/kibana_react/public';
 
 function renderOption(option, searchValue, contentClassName) {
   return (
-    <span className={contentClassName}>
-      <FieldIcon type={option.value.type} size="m" useColor />
-      &nbsp;
-      <EuiHighlight search={searchValue}>{option.label}</EuiHighlight>
-    </span>
+    <EuiFlexGroup className={contentClassName} gutterSize="s" alignItems="center">
+      <EuiFlexItem grow={null}>
```

Comment:
Same here?

Reply:
These values aren't technically wrong, so I'm just going to allow them.